whois-update/src/main/java/net/ripe/db/whois/update/handler/validator/organisation/OrgNameAndCountryAttrValidator.java

```diff
@@ -84,14 +93,11 @@ private boolean isReferencedByRsMaintainedResource(final RpslObject rpslObject)
         }
         return false;
     }
+    private boolean isAttributeModified(final AttributeType attrType, final RpslObject originalObject, final RpslObject updatedObject) {
+        final String originalAttrValue = originalObject.containsAttribute(attrType) ? originalObject.getValueForAttribute(attrType).toString() : null;
+        final String updatedAttrValue = updatedObject.containsAttribute(attrType) ? updatedObject.getValueForAttribute(attrType).toString() : null;
 
-    private boolean orgNameDidntChange(final RpslObject originalObject, final RpslObject updatedObject) {
-        final CIString originalOrgName = originalObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
-        final CIString updatedOrgName = updatedObject.getValueOrNullForAttribute(AttributeType.ORG_NAME);
-
-        return (originalOrgName != null) &&
-                (updatedOrgName != null) &&
-                (Objects.equals(originalOrgName.toString(), updatedOrgName.toString()));
+        return !StringUtils.equals(originalAttrValue, updatedAttrValue);
```

Comment:
Using Strings fails where the *case* of the value changes (e.g. NL to nl), but this *shouldn't* be considered a change, the value is case-insensitive.

Reply:
I think the old code had this issue also